python/spacewalk/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -525,6 +526,13 @@ def _get_mirror_list(self, repo, url):
         mirrorlist_path = os.path.join(repo.root, 'mirrorlist.txt')
         returnlist = []
         content = []
+        # If page not plaintext or xml, is not a valid mirrorlist or metalink,
+        # so continue without it.
+        with request.urlopen(url) as webpage:
```

Comment:
It seems in this case it will ignore HTTP_PROXY settings set with web UI (rhn.conf).